projects/plugins/jetpack/extensions/blocks/one-payment/variations.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * The different payment blocks that can be chosen.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { settings as DonationSettings } from '../donations';
+import { settings as RecurringPaymentSettings } from '../recurring-payments';
+
+const variations = [
+	{
+		name: 'onepayment-donations',
+		title: DonationSettings.title,
+		description: DonationSettings.description,
+		icon: DonationSettings.icon.src,
```

Comment:
With `DonationSettings.icon` The one payment block crashes when selected and I get this:

> Objects are not valid as a React child (found: object with keys {foreground, src}). If you meant to render a collection of children, use an array instead. 

so I have to go for the src, the impact is that the foreground colour (i.e. jetpack green on a JP site) isn't used.